Makefile

```diff
@@ -20,4 +20,7 @@ lint: $(CONFIG)
 test:
 	bundle exec rspec
 
+build_nr_scripts:
```

Comment:
since this is a pretty direct 1:1 translation, what if we used makefiles for dependency stuff?

```
new_relic_scripts_out/totp_sign_in.js: new_relic_scripts/totp_sign_in.js.erb .env
  ./bin/build_nr_scripts.sh $< > $@
```

Reply:
I could have the Makefile magic variables wrong, I just thought they'd be nice, but the gist is the same